package.json

```diff
@@ -29,7 +29,7 @@
     "postinstall": "node node_modules/bower/bin/bower install"
   },
   "engines": {
-    "node": "~0.10.0"
+    "node": "0.10.x"
```

Comment:
Equivalent, just switching to the prevailing style. I assume we want to do more thorough testing before adding or switching to `"0.12.x"`.